calculator.rb

```diff
@@ -0,0 +1,70 @@
+puts "Welcome to the Calculator!"
+
+puts "What type of math do you need doing? Please choose from this list and put the name or symbol:"
+puts "Add (+)"
+puts "Subtract (-)"
+puts "Multiply (*)"
+puts "Divide (/)"
+
+valid_operator_array = ["add","subtract","multiply","divide","+","-","*","/"]
+operator = gets.chomp.downcase
+
+until valid_operator_array.include?(operator)
+  puts "Please put in a valid input.(Add, Subtract, Multiply, Divide, +,- ,* ,/ )"
+  operator = gets.chomp.downcase
+end
+
+def valid_number(number)
+  alphabet_array = [*'a'..'z', *'A'..'Z']  # https://stackoverflow.com/questions/4846853/generate-array-of-all-letters-and-digits
+  operator_array = ["!","@","#","$","%","^","&","(",")","[","]","{","}","?","_","``","~","|","<",">",",",";",":","=","+","-","*","/"]
+  if number == 0 && number.to_i.to_s == number #for when people truly put in 0
+    return "yes"
```

Comment:
It would probably be better to return `true` if it is a number, and `false` if it isn't.